src/main/java/ch/wisv/areafiftylan/service/MailServiceImpl.java

```diff
@@ -92,4 +92,10 @@ public void sendPasswordResetMail(User user, String url) throws MessagingExcepti
         String message = "Please click on the following link to reset your password: " + url;
         sendMail(user.getEmail(), user.getUsername(), null, "Password reset requested", message);
     }
+
+    @Override
+    public void sendTicketTransferMail(User sender, User receiver, String url) throws MessagingException {
+        String message = "The user \"" + sender.getUsername() + "\" has send you a ticket for AreaFiftyLAN! To accept this ticket please click on the following link: " + url;
```

Comment:
I'd change this to include the full name of the sender. Tickets were bought, so a filled in profile is guaranteed. The term "user" isn't really something you want to say. They're people! :stuck_out_tongue_winking_eye: